tests/config/test_functional_config_loading.py

```diff
@@ -78,9 +79,13 @@ def test_functional_config_loading(
     expected_loaded_configuration = get_expected_configuration(
         configuration_path, default_configuration
     )
-    mock_exit, _, runner = run_using_a_configuration_file(
-        configuration_path, file_to_lint_path
-    )
+    with warnings.catch_warnings():
```

Comment:
does `pytest.warns` both do the ignoring and the asserting? This looks like it just does the ignoring?

Reply:
Yeah let's not sink a lot of time into this but was just honestly wondering if it's easy as adding `pytest.warns`, but if not, then not!

Reply:
No since `pytest` will complain about all functional tests that do not have `MASTER`. I just checked again manually and your suggestion works as intended. Thanks!